docs/source/traitsui_user_manual/testing.rst

```diff
@@ -52,6 +52,6 @@ Technical References
    :maxdepth: 1
 
    testing/references/examples
-   API references <../api/traitsui.testing>
+   API References <../api/traitsui.testing>
```

Comment:
Ah I meant make it singular ("Reference") as I think that is what traits etc. call it.  However, tbh I'm not really sure what is grammatically correct / sounds better, it was simply a comment on consistency

Reply:
Oops, I overlooked that. Yes you are right. Let's go with what is already there.

Reply:
Argh... The "Technical References" should be "Technical Reference" too. Will fix in a separate PR... (very tempted to make it drive-by in the next one)

Reply:
Ah my mistake I missed that, I think a drive by is fine